CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## master
 
+- Support passing rewriters to CLI. ([@sl4vr][])
```

Comment:
👍 
Could you please also add the actual link to the bottom of this file? (Right now it's not working, cause shortcut is not defined)

Reply:
Oops I thought it's Github feature or something 🙄